index.bs

```diff
@@ -10891,7 +10892,13 @@ communication (asynchronous) between
 {{AudioWorkletNode}} and
 {{AudioWorkletProcessor}}. This node does not use any output.
 
-<xmp line-numbers class="example" highlight="js" title="VUMeterNode - Global Scope (vumeternode.js)">
+<div class="correction" id="c2359">
+  <span class="marker">Candidate Correction
+    <a href="https://github.com/WebAudio/web-audio-api/issues/2359">Issue 2359.</a>
+  </span>
+  Fix typo in code; semi-colon is incorrect.
+</div>
+<pre line-numbers class="example" highlight="js" title="VUMeterNode - Global Scope (vumeternode.js)">
```

Comment:
Needed to change xmp to pre so that the del markup works to show what was deleted.